pom.xml

```diff
@@ -530,7 +361,6 @@ THE SOFTWARE.
                     </execution>
                 </executions>
                 <configuration>
-                    <pluginFirstClassLoader>true</pluginFirstClassLoader>
```

Comment:
we do not need this, we are safe with `maskClasses` now after removing other conflicting libraries